static/scss/general_styles.scss

```diff
@@ -0,0 +1,47 @@
+// General Style Rules
+
+body {
+  background-color: $page-background;
+  font-family: 'Roboto', helvetica, arial, sans-serif !important;
+  line-height: 1.4em;
+  }
+
+ .mdl-card {
+    padding: 20px 24px 30px;
+    margin: 0 0 22px 0;
+    min-height: 150px;
+    width: 100%;
```

Comment:
this is what's causing the issue with the user chip - I think we should remove it.

I took out this line on my machine and it looks like the only things which are affected negatively are the cards on the `/dashboard` - could we add a `full-width` class to those and then scope the `width: 100%` rule to `.mdl-card.full-width`?